src/ecrc-frontend/src/components/base/orgValidationText/OrgValidationText.js

```diff
@@ -14,52 +14,61 @@ export default function OrgValidationText({
+        enables you to easily apply for a criminal record check under the{" "}
         <a href="http://www.bclaws.ca/EPLibraries/bclaws_new/document/ID/freeside/00_96086_01">
           Criminal Records Review Act (CRRA).
         </a>
       </p>
       <div>
         <div>
-          <h3>{"I'm ready"}</h3>
+          <br />
+          <h3>{"How do I submit a CRC request online?"}</h3>
           <p>
             To submit an online request for a criminal record check, you must:
           </p>
           <ul className="bodyList">
             <li>{"Be at least 12 years of age as of today's date"}</li>
-            <li>Have an access code provided by your organization</li>
             <li>
-              Have your identity verified through an Electronic Verification
-              Identity (EIV) process.
-              <p style={{ fontStyle: "italic" }}>
-                <strong> NOTE. </strong>Not all individuals will be able to use
-                an EIV process and may be required to submit the request to a
-                criminal record check through a manual process.
-              </p>
+              Have your identity verified by using your{" "}
+              <a href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card">
+                BC Services Card
+              </a>{" "}
+              (BCSC). The BCSC provides secure access to provincial government
+              online services. If you don&apos;t have a BCSC, an alternative
+              option may be available for you.
             </li>
+            <li>Have an access code provided by your organization</li>
           </ul>
-          <div style={{ maxWidth: "350px", float: "left" }}>
-            <p>
-              Enter the access code provided by your organization. An access
-              code is required to proceed with the online submission:
-            </p>
-          </div>
+        </div>
+
+        <div>
+          <br />
+          <h3>{"I'm ready"}</h3>
           <div style={{ float: "left" }}>
             <div>
               <TextInput textInput={textInput} onChange={onChange} />
             </div>
-            <div>
-              <strong>Access code must be entered to proceed *</strong>
-            </div>
-            <p />
-            <div style={{ float: "right" }}>
-              <Button button={button} onClick={onClick} />
+            <div style={{ maxWidth: "350px" }}>
+              <p style={{ width: "400px" }}>
+                Enter the access code provided by your organization. An access
+                code is required to proceed with the online submission
+              </p>
             </div>
           </div>
+          <div style={{ float: "left" }}>
+            <span id="asterisk" className="mandatory">
+              *
+            </span>
+          </div>
+          <div style={{ float: "right" }}>
+            <Button button={button} onClick={onClick} />
+          </div>
```

Comment:
I noticed in the screenshot that the continue button is way over on the far right of the screen. Should it be a little closer to the access code text field?

Reply:
I think it is meant to be at the right corner of the page (in terms of where the page content ends) - also this should look less weird when captcha is added in

Reply:
Okay sounds good, I'll take another look after we add the captcha.